libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -62,6 +59,7 @@
     private static final String KEY_REQUESTED_EMAIL = "KEY_REQUESTED_EMAIL";
     private static final String LOG_TAG = LoginEmailFragment.class.getSimpleName();
     private static final int GOOGLE_API_CLIENT_ID = 1001;
+    private static final int EMAIL_CREDENTIALS_REQUEST_CODE = 25100;
```

Comment:
This was moved because `RequestCodes` is not visible to the login library project and wouldn't really make sense in a shared library like utils either.

Reply:
That makes sense.  Thanks for the clarification!